### ISSUE TITLE
Uninstalled. Now brightness won't go to 100%!!

### ISSUE THREAD

**Issue by m22tanker** [blackholeearth/Win10_BrightnessSlider#69]
Deleted the exe file and got rid of brightness slider. Now my display won't go to 100% Display is very dim. Is there a fix? Slider in Night Light settings is at full.

**Comment by gnarlex** [comment]
Already tried reinstalling and readjusting slider?

**Comment by m22tanker** [comment]
No, I didn't think of that. BTW, what is the proper procedure to uninstall?

**Comment by Nu11u5** [comment]
The app communicates directly with the monitor to change the settings in the monitor's OSD menu.  It has nothing to do with nightlight.

Check the brightness setting in your monitor's OSD menu.
